spec/services/github_service_spec.rb

```diff
@@ -59,35 +59,40 @@
         end
       end
 
-      describe '#review_request_removed' do
-        let!(:pull_request) { create :pull_request, github_id: payload['pull_request']['id'] }
-        let!(:reviewer) do
-          create :user, github_id: payload['requested_reviewer']['id']
-        end
-        let!(:second_reviewer) do
-          create :user
-        end
-        let!(:owner) do
-          create :user, github_id: payload['pull_request']['user']['id']
-        end
-        let!(:review_request) do
-          create :review_request,
-                 owner: owner,
-                 reviewer: reviewer,
-                 pull_request: pull_request
-        end
+      describe 'when the action is review_request_removed' do
```

Comment:
You could declare an `action` variable under this describe:
`let(:action) { 'review_request_removed' }`
So that you can avoid executing `change_action_to('review_request_removed')`

Reply:
Done it.
Thanks!